README.md

```diff
@@ -22,21 +22,529 @@ A Spezi framework that provides a common set of SwiftUI views and related functi
+
+Easily share content with the system share sheet:
+
+```swift
+import SpeziViews
+import SwiftUI
+
+struct ShareView: View {
+    @State private var itemToShare: ShareSheetInput?
+    
+    var body: some View {
+        VStack {
+            Button("Share Text") {
+                itemToShare = ShareSheetInput("Hello World!")
+            }
+            
+            Button("Share Image") {
+                if let image = UIImage(systemName: "star") {
+                    itemToShare = ShareSheetInput(image)
+                }
+            }
+        }
+        .shareSheet(item: $itemToShare)
+    }
+}
+```
+
+#### Skeleton Loading
+
+Add skeleton loading animations:
+
+```swift
+import SpeziViews
+import SwiftUI
+
+struct LoadingView: View {
+    var body: some View {
+        VStack {
+            RoundedRectangle(cornerRadius: 10)
+                .fill(Color(.systemGray4))
+                .frame(height: 100)
+                .skeletonLoading(replicationCount: 5, repeatInterval: 1.5, spacing: 16)
+        }
+        .padding()
+    }
+}
+```
+
+### SpeziPersonalInfo - Personal Information
+
+Handle name input with proper formatting:
+
+```swift
+import SpeziPersonalInfo
+import SwiftUI
+
+struct ProfileSetupView: View {
+    @State private var name = PersonNameComponents()
+    
+    var body: some View {
+        Form {
+            Section("Personal Information") {
+                Grid(horizontalSpacing: 15) {
+                    NameFieldRow("First", name: $name, for: \.givenName) {
+                        Text("Enter your first name")
+                    }
+                    
+                    Divider()
+                        .gridCellUnsizedAxes(.horizontal)
+                    
+                    NameFieldRow("Middle", name: $name, for: \.middleName) {
+                        Text("Enter your middle name (optional)")
+                    }
+                    
+                    Divider()
+                        .gridCellUnsizedAxes(.horizontal)
+                    
+                    NameFieldRow("Last", name: $name, for: \.familyName) {
+                        Text("Enter your last name")
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+### SpeziValidation - Input Validation
+
+Add real-time validation to forms:
+
+```swift
+import SpeziValidation
+import SwiftUI
+
+struct SignupView: View {
+    @State private var email = ""
+    @State private var password = ""
+    @State private var username = ""
+    
+    @ValidationState private var validation
+    
+    var body: some View {
+        Form {
+            Section("Account Details") {
+                VerifiableTextField("Email", text: $email)
+                    .validate(input: email, rules: .minimalEmail)
+                    .textInputAutocapitalization(.never)
+                    .autocorrectionDisabled()
+                
+                VerifiableTextField("Password", text: $password, type: .secure)
```

Comment:
If we have this as an example for best practices we should probably also add content annotations to all text fields to better support autofill.